Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM registry.redhat.io/ubi8/go-toolset
+
+WORKDIR /go/src/app
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+COPY . .
+
+USER root
+
+RUN go build -o ws-gateway ./cmd/gateway/main.go
```

Comment:
Is this supposed to build the image that gets deployed for the gateway and switch?  If so, we need to include the gateway, job-receiver, response_consumer, connection_util, connection_cleaner.